.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -76,6 +76,7 @@ jobs:
       curr_branch: ${{ github.head_ref || github.ref_name }}
       curr_ref_type: ${{ github.ref_type }}
 
+  # NB: Keep this in sync with trunk.yml
```

Comment:
@suo added `sync-tag:` and linter in https://github.com/pytorch/pytorch/blob/main/tools/linter/adapters/workflow_consistency_linter.py a while back, would be nice to use here